Fixed #34609 -- Deprecated calling format_html() without arguments. (#16948)
django/django: docs/internals/deprecation.txt

* The default scheme for ``forms.URLField`` will change from ``"http"`` to
  ``"https"``.

> * Support for calling format_html() without args or kwargs will be removed.

Reviewer: This should be documented in `format_html()` docs and release notes.

> * Support for calling format_html() without args or kwargs will be removed.

Reviewer 2: Also:

```suggestion
* Support for calling ``format_html()`` without any arguments will be removed.
```
